x-pack/platform/plugins/shared/onechat/kibana.jsonc

```diff
@@ -8,31 +8,24 @@
     "id": "onechat",
     "server": true,
     "browser": true,
-    "configPath": [
-      "xpack",
-      "onechat"
-    ],
+    "configPath": ["xpack", "onechat"],
     "requiredPlugins": [
       "data",
       "dataViews",
-      "uiActions",
+      "embeddable",
       "esql",
       "features",
       "inference",
       "kibanaUtils",
       "lens",
       "management",
+      "maps",
       "share",
+      "uiActions",
```

Comment:
I have PTSD about cyclic dependencies, so I will ask for all newly introduced plugin deps:

no risk we're depending on a plugin which may later want to use agent builder, right? `embeddable` and `maps` are both low-level plugins without an actual app / logic which may want to depend on agent builder later?

Reply:
maps is an app, so is lens (which was already there for the sake of visualize_esql)... but we need to get their embeddable somehow in order to render.

uiActions and embeddable look fine.